charts/gardener/gardenlet/charts/runtime/templates/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
 {{ toYaml .Values.global.gardenlet.podLabels | indent 8 }}
         {{- end }}
     spec:
-      priorityClassName: gardenlet
+      priorityClassName: gardener-system-critical
```

Comment:
I think this `PriorityClass` should be part of this Helm chart (`gardenlet`) and removed from `seedsystem` since `gardenlet` is typically the first component deployed to a new seed cluster. This is probably also why the e2e-test is currently failing.

Reply:
Make sense and yes, the current reason for the e2e test failure is indeed that. However, should we still keep the `PriorityClass` in the `seedsystem` for integrity and to be recreated by the grm in case someone deletes it by mistake?

Reply:
No, I would remove it from there to prevent duplication. Nobody other than admins/operators can delete it, but that's not different compared to any other resource in the seed

Reply:
@rfranzke now there is no one to deploy the `gardener-system-critical` `PriorityClass` when running the local setup. We could probably add it to the `start-gardenlet` script wdyt?

Reply:
Yeah, sounds reasonable